mysql.go

```diff
@@ -245,12 +191,227 @@ func (driver *Driver) Versions() (file.Versions, error) {
 	return versions, err
 }
 
-// Execute a SQL statement
-func (driver *Driver) Execute(statement string) error {
-	_, err := driver.db.Exec(statement)
+// Lock schema_migrations table
+func (drv *Driver) Lock() error {
+	// NOTE: versionTx will be implicitly committed by LOCK TABLES
+	// the only reason we need it is to get exclusive db connection.
+	// TODO: go1.9 has DB.Conn() which returns exclusive connection, use it when time comes.
```

Comment:
Isn't the time already?

Reply:
We don't use 1.9 in production yet,  waiting some time before upgrading minor version of Go is quite popular practice.  I built custom migrations appliance on top of migrate package. Moving it to DB.Conn() is not a big deal, let's wait a while.

Reply:
ok fine